src/project1.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>My Webpage new</title>
+    <link rel="stylesheet" href="C:/Users/Deeksha/Desktop/project/project1.css">
```

Comment:
never put paths that link to your system path, always make sure all paths are in project level only.

Reply:
you can always find relative path by right click on the css / js file and get relative path option

Reply:
Yes understood.